src/main/scala/it/pagopa/interop/apigateway/service/impl/AuthorizationManagementServiceImpl.scala

```diff
@@ -26,22 +27,10 @@ class AuthorizationManagementServiceImpl(invoker: AuthorizationManagementInvoker
       request = api.getClient(xCorrelationId = correlationId, clientId = clientId, xForwardedFor = ip)(
         BearerToken(bearerToken)
       )
-      result <- invoker.invoke(request, s"Retrieving client by id = $clientId", handleCommonErrors(s"client $clientId"))
+      result <- invoker
+        .invoke(request, s"Retrieving client by id = $clientId")
+        .adaptError { case err: ApiError[_] if err.code == 404 => ClientNotFound(clientId) }
```

Comment:
same as [here](https://github.com/pagopa/interop-be-authorization-server/pull/74/files#r1050531862)

Reply:
this is a `Try`, it should be safe, right?